BREAKING.md

```diff
@@ -29,6 +30,9 @@ Remove deprecated `shipAssert` `debugAssert` `logException` `logGenericError` in
 ### fileOverwrittenInStorage
 Please use `DriverErrorType.fileOverwrittenInStorage` instead of `OdspErrorType.epochVersionMismatch`
 
+### absolutePath use in IFluidHandle is deprecated
+Rather than retrieving the absolute path, ostensibly to be stored, one should instead store the handle itself. To load, first retrieve the handle and then call `get` on it to get the actual object.
+
```

Comment:
Consider adding that it is assumed that container is responsible for mapping external URIs to internal objects via some kind of map external URI -> handle, and requesting resolved object with remaining tail of external URI.
I.e. 
mapping: /a -> some handle
Logic: /a/b/c -> handle -> object -> request("/b/c");

Reply:
Added